framework/wazuh/core/cluster/local_server.py

```diff
@@ -349,9 +349,8 @@ def __init__(self, node: server.AbstractServer, **kwargs):
         super().__init__(node=node, **kwargs)

```

Comment:
This constructor accepts a node of type `server.AbstractServer`, but the docstrings say it accepts `AbstractServer, AbstractClientManager`, we should change the type hint.

https://github.com/wazuh/wazuh/blob/cf1a7af9cbaacc8539f48343663ca65b37601600/framework/wazuh/core/cluster/local_server.py#L339-L353